admin/app/templates/login.hbs

```diff
@@ -1,6 +1,10 @@
 <div class="login-page">
   <main class="login-page__main">
-    <LoginForm />
+    <LoginForm
+      @isUnknownErrorOccurs={{this.isUnknownErrorOccurs}}
+      @isUserShouldCreateAnAccount={{this.isUserShouldCreateAnAccount}}
+      @isUserShouldRequestAccess={{this.isUserShouldRequestAccess}}
```

Comment:
Les noms `isUnknownErrorOccurs`, `isUserShouldCreateAnAccount`, `isUserShouldRequestAccess` sont vraiment difficiles à lire 🙁 Je comprends le soin qui a été apporté pour les faire commencer par des `is` mais là ça parait vraiment inadapté. Voici des suggestions de noms qui contiennent toujours des verbes qui ne sont pas des verbes d'action : 

```suggestion
      @unknownErrorHasOccured={{this.unknownErrorHasOccured}}
      @userShouldCreateAnAccount={{this.userShouldCreateAnAccount}}
      @userShouldRequestAccess={{this.userShouldRequestAccess}}
```